ddl/get.lua

```diff
@@ -66,11 +67,24 @@ local function get_metadata(space_name, metadata_name)
 end
 
 local function get_sharding_func(space_name)
-    local record = get_metadata(space_name, "sharding_func")
```

Comment:
We have a function `get_metadata()` used in function `get_sharding_key()` and `get_sharding_func()`. After your patch, it will be used in `get_sharding_key()` only. I remember that you implement a cache for sharding *function*. But it is a bit inconsistent: `get_sharding_func` uses a cache with sharding metadata and while `get_sharding_key` does not.

Perhaps we need to replace `get_metadata()` with `cache.internal.get` everywhere and remove `get_metadata` at all.

Reply:
Your point is valid, but I'm afraid that if we'll going to implement caching of the sharding keys here (or even stubs for it), we'll expand the scope of the task too much.